src/System.AppContext/ref/System.AppContext.csproj

```diff
@@ -4,8 +4,8 @@
   <PropertyGroup>
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <PackageTargetFramework>netstandard1.5</PackageTargetFramework>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
+    <PackageTargetFramework>netstandard1.6</PackageTargetFramework>
```

Comment:
So you added two apis:
AppContext.TargetFrameworkName 
AppContext.GetData

Suppose TargetFrameworkName could be supported on .NET 4.6.2.  In that case you could have separate build that targets netstandard1.5 that exposes just this, and a build that targets netstandard1.6 and exposes GetData.

If TargetFrameworkName is only supported in .NET 4.6.3 and not 4.6.2 then you don't have this option.